packages/devtools/src/schema/type-definitions.ts

```diff
@@ -35,8 +35,9 @@ directive @documentReference(model: String!) on FIELD_DEFINITION
 # Relation views
 
 directive @relationDocument(property: String!) on FIELD_DEFINITION
-directive @relationFrom(model: String!, property: String!) on FIELD_DEFINITION
+directive @relationFrom(model: String, property: String!) on FIELD_DEFINITION
```

Comment:
Why don't we need the model here? Wouldnt this throw 'Missing model for ${source} relation view'?

Reply:
I updated the logic to use the model defined in as the field type instead, so the `model` argument is no longer needed, I only kept it to avoid throwing errors in schemas that keep using it.